data_updates/Python/iati_transaction_spec.py

```diff
@@ -1051,14 +1060,17 @@ def flatten_activities(self, root):
                     x_transaction_value_usd = ""
                     x_reporting_org_type = recode_if_not_none(reporting_org_type_code, self.dictionaries["organisation_type"])
                     x_transaction_type = recode_if_not_none(transaction_type_code, self.dictionaries["transaction_type"])
+                    x_donor_transaction_type = recode_if_not_none(transaction_type_code, self.dictionaries["donor_transaction_type"])
```

Comment:
This might need an additional check before to avoid final value being `None`. Something like:
```
if transaction_type_code in self.dictionaries["donor_transaction_type"]:
    x_donor_transaction_type = recode_if_not_none(transaction_type_code, self.dictionaries["donor_transaction_type"])
```